doc/user/content/get-started/install-materialize.md

```diff
@@ -0,0 +1,145 @@
+and run Materilize locally in a Docker container.  The Docker image:
+
+- Includes the [Quickstart](/get-started/quickstart) data source
+
+- Supports connecting your data sources
+
+- Provides [community support via
+  Slack](https://materializecommunity.slack.com/join/shared_invite/zt-2bad5ce4i-ZsiPWI5jd7Q9pRDGYj3dkw#/shared-invite/email)
+
+To more fully evaluate
+Materialize and its features, Materialize offers a [free trial
+account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).
+
+{{< warning >}}
+
+The Docker image is for <redb> testing/evaluation purposes only. It is not
+suitable for production deployments</redb>.  See also [Limitations](#limitations).
+
+{{</ warning >}}
+
+### License and Privacy Policy
+
+- Use of the Docker image is subject to Materialize's [BSL
+  License](https://github.com/MaterializeInc/materialize/blob/main/LICENSE).
+
+- By downloading the Docker image, you are agreeing to Materialize's [privacy
+  policy](https://materialize.com/privacy-policy/).
+
+### Prerequisites
+
+- Docker. If [Docker](https://www.docker.com/) is not installed, refer to its
+[official documentation](https://docs.docker.com/get-docker/) to install.
+
+### Run Materialize in a Docker container
+
+{{< note >}}
+
+- Use of the Docker image is subject to Materialize's [BSL
+   License](https://github.com/MaterializeInc/materialize/blob/main/LICENSE).
+
+- By downloading the Docker image, you are agreeing to Materialize's [privacy
+   policy](https://materialize.com/privacy-policy/).
+
+{{</ note >}}
+
+1. In a terminal, issue the following command to run a Docker container from the
+   Materialize image. The command downloads the Materialize image, if one has
+   not been already downloaded.
+
+   ```sh
+   docker run -v mzdata:/mzdata -p 6875:6875 -p 6876:6876 materialize/materialized
+   ```
+
+   When running locally:
+
+   - Materialize logs are emitted to the `stderr`.
+   - The SQL interface is available on port `6875`.
+   - The HTTP interface is available on port `6876`.
+   - A default user `materialize` is created.
+   - A default database `materialize` is created.
+
+1. Connect to Materialize using your preferred SQL client, using the following
+   connection field values:
+
+   | Field    | Value         |
+   |----------|---------------|
+   | Server   | `localhost`   |
+   | Database | `materialize` |
+   | Port     | `6875`        |
+   | Username | `materialize` |
+
+   For example, if using `psql`
+
+   ```sh
+   psql postgres://materialize@localhost:6875/materialize
+   ```
+
+1. Once connected, you can get started with the
+   [Quickstart](/get-started/quickstart). Where a command for the Materialize
+   console can differ from other SQL clients, both versions of the command are
+   presented.
+
+### Next steps
+
+To get started ingesting your own data from an external system like Kafka, MySQL
+or PostgreSQL, check the documentation for [sources](/sql/create-source/).
```

Comment:
@ggnall, re-stating that we should provided Docker Compose files that come pre-baked with these services, at some point. Otherwise, we'll spend 99.9% of our support time debugging Docker networking issues.

Reply:
good call. noting!